wa/workloads/motionmark/__init__.py

```diff
@@ -0,0 +1,81 @@
+#    Copyright 2014-2019 ARM Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+import os
+import re
+
+from wa import UiautoWorkload, Parameter
+from wa.framework.exception import ValidationError, WorkloadError
+from wa.utils.types import list_of_strs
+from wa.utils.misc import unique
+
+
+class Motionmark(UiautoWorkload):
+
+    name = 'MotionMark'
+
+    description = '''
+    A workload to execute the motionmark web based benchmark
+
+    MotionMark is a graphics benchmark that measures a browser capability to animate complex scenes at a target frame rate
+
+    Test description:
+    1. Open browser application
+    2. Navigate to the motionmark website - http://browserbench.org/motionmark/
```

Comment:
Please change this url to the correct capitalisation of the destination site `http://browserbench.org/MotionMark/`